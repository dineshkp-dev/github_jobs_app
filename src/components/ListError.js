import React from 'react';
import '../styles/listerror.css';

function ListError(props) {
  const loadingText = (props && props.text) || 'No listings available.';
  return (
    <div className='listerror-container'>
      <h3 className='listerror-title'>No listing found</h3>
      <div className='listerror-text'>{loadingText}</div>
    </div>
  );
}

export default ListError;
