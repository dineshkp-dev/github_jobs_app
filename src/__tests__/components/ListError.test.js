import ListError from '../../components/ListError';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

describe('ListError snapshot test', () => {
  test('renders', () => {
    const { container } = render(<ListError />);
    expect(container).toMatchSnapshot();
  });
});
