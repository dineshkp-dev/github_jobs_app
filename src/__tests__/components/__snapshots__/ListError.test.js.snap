// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListError snapshot test renders 1`] = `
<div>
  <div
    class="listerror-container"
  >
    <h3
      class="listerror-title"
    >
      No listing found
    </h3>
    <div
      class="listerror-text"
    >
      No listings available.
    </div>
  </div>
</div>
`;
